modules/registry-operator-config-resources-storage-credentials.adoc

```diff
@@ -21,7 +21,7 @@ credentials used by the Operator, if default credentials were found.
 +
 [source,terminal]
 ----
-$ oc create secret generic image-registry-private-configuration-user --from-file=KEY1=value1 --from-literal=KEY2=value2 --namespace openshift-image-registry
+$ oc create secret generic image-registry-private-configuration-user--from-literal=KEY1=value1 --from-literal=KEY2=value2 --namespace openshift-image-registry
```

Comment:
Need add space between `image-registry-private-configuration-user` and `--from-literal=KEY1=value1`

Reply:
Thank you! How does it look now? @xiuwang